.drone.yml

```diff
@@ -7635,241 +7238,25 @@ volumes:
   temp: {}
 
 ---
-################################################
-# Generated using dronegen, do not edit by hand!
-# Use 'make dronegen' to update.
-# Generated at dronegen/promote.go (main.buildDockerPromotionPipelineECR)
-################################################
-
 kind: pipeline
 type: kubernetes
-name: promote-docker-ecr
+name: promote-build
```

Comment:
The re-ordering creates a lot of noise.

Reply:
This isn't a reordering strictly speaking. It's a complete removal of the `promote-docker-ecr` and `promote-docker-quay` pipelines